tests/ci/cdk/cdk/aws_lc_github_ci_stack.py

```diff
@@ -35,18 +35,37 @@ def __init__(self,
             ],
             webhook_triggers_batch_build=True)
 
+        # Define a IAM role for accessing build resources
+        log_group = logs.LogGroup(self, id="{}-public-logs".format(id))
+        code_build_cloudwatch_logs_policy = iam.PolicyDocument.from_json(
+            code_build_cloudwatch_logs_policy_in_json([log_group.log_group_name])
```

Comment:
You should just be able to pass in `log_group.log_group_arn` (see [API](https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_logs/LogGroup.html#aws_cdk.aws_logs.LogGroup.log_group_arn)) without needing to format/template the ARN yourself.

Reply:
Thanks. Fixed in latest commit.